Python/Product/Django/Project/DjangoProject.cs

```diff
@@ -57,6 +58,20 @@ partial class DjangoProject :
         public DjangoProject() {
         }
 
+        public void Dispose() {
```

Comment:
Who actually calls this Dispose method?  It's also weird to see the full Dispose pattern with the bool disposing parameter when this doesn't a finalizer that calls it with Dispose(false).